client/verta/verta/_pipeline_orchestrator/_step_handler.py

```diff
@@ -0,0 +1,165 @@
+# -*- coding: utf-8 -*-
+
+import abc
+import json
+from typing import Any, Dict, Iterable, List, Optional, Set, Type
+
+from verta._internal_utils import _utils, http_session
+from verta.registry.entities import RegisteredModelVersion
+from verta.tracking.entities._entity import _MODEL_ARTIFACTS_ATTR_KEY
+
+
+class _StepHandlerBase(abc.ABC):
```

Comment:
Basically, there's

- `ModelContainerStepHandler` to route predictions to a deployed model container
- `ModelObjectStepHandler` to route predictions to a locally-instantiated model object

These subclasses implement that logic to route predictions. This base class (`_StepHandlerBase`) doesn't do much.